packages/qdrant-loader/src/qdrant_loader/connectors/confluence/connector.py

```
@@ -792,162 +804,256 @@ def _clean_html(self, html: str) -> str:
-                    logger.error(
-                        f"Failed to fetch content from space {self.config.space_key}: {e!s}"
-                    )
-                    raise
+            except (requests.exceptions.RequestException, ValueError) as e:
+                logger.error(
+                    f"Failed to fetch content from space {self.config.space_key}: {e!s}"
+                )
+                raise
+
+    async def stream_documents(
+        self, since: datetime | None = None
```

Ruff (0.15.20)

[warning] Unused method argument: `since`

(ARG002)